templating/templating.py

```diff
@@ -8,17 +8,20 @@
 import requests
 
 from flask import Flask, request
-from jinja2 import Environment, FileSystemLoader, TemplateNotFound
+from jinja2 import Environment, FileSystemLoader, TemplateNotFound, TemplateError
 
 endpoints = "read/networks read/oplog read/snmp read/switches-management public/distro-tree public/config public/dhcp public/dhcp-summary public/ping public/switches public/switch-state".split()
```

Comment:
```suggestion
endpoints = ["read/networks", "read/oplog", "read/snmp", "read/switches-management", "public/distro-tree",
             "public/config", "public/dhcp", "public/dhcp-summary", "public/ping", "public/switches",
             "public/switch-state"]
```
Just for readability.